### DISCUSSION TITLE
memory problem

### DISCUSSION THREAD

**Discussion by Dau-Duc-Hai** [pimcore/pimcore#12063] [Q&A / Support]
In debug mode (open and look at code then system stop working with following alert)
Fatal error: Allowed memory size of 268435456 bytes exhausted (tried to allocate 20480 bytes) in /var/www/html/dam/vendor/doctrine/dbal/lib/Doctrine/DBAL/SQLParserUtils.php on line 154

Fatal error: Allowed memory size of 268435456 bytes exhausted (tried to allocate 65536 bytes) in /var/www/html/dam/vendor/symfony/error-handler/DebugClassLoader.php on line 284

-- Please advice what happen? thank you

**Comment by ghost** [comment]
@Dau-Duc-Hai  increase the limits on yr php.in  max_input_vars, memory_limit, post_max_size

**Reply by Dau-Duc-Hai** [reply]
Thank you, 
I think it related to 10.4 bugs. I will update to 10.4.1 and let you know if was problem.
